Formula/grin-wallet.rb

```diff
@@ -14,6 +14,15 @@ class GrinWallet < Formula
   depends_on "llvm" => :build # for libclang
   depends_on "rust" => :build
 
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "openssl@1.1"
+  end
+
+  # Fix build on Rust 1.53.0+. Remove in the next release.
+  # Backport of https://github.com/mimblewimble/grin-wallet/pull/621
+  patch :DATA
```

Comment:
Can we just download the `Cargo.lock` off the latest commit and replace the one in the tarball?

Reply:
Maybe. I wasn't sure if some dependency changes would pull in incompatible APIs since most of the other Cargo.lock/toml changes were updated along with other Rust code changes.

Might as well give it a try.